Library/Formula/rpm.rb

```diff
@@ -15,30 +15,29 @@ def ext
 
 class Rpm < Formula
   homepage 'http://www.rpm5.org/'
-  url 'http://rpm5.org/files/rpm/rpm-5.4/rpm-5.4.14-0.20131024.src.rpm',
+  url 'http://rpm5.org/files/rpm/rpm-5.4/rpm-5.4.15-0.20140824.src.rpm',
       :using => RpmDownloadStrategy
-  version '5.4.14'
-  sha1 'ea1a5f073ba4923d32f98b4e95a3f2555824f22c'
+  version '5.4.15'
+  sha1 '5e94f4679759c36ce76a4847401b22b660a97227'
 
   depends_on 'berkeley-db'
   depends_on 'libmagic'
   depends_on 'popt'
-  depends_on 'beecrypt'
   depends_on 'libtasn1'
-  depends_on 'neon'
   depends_on 'gettext'
   depends_on 'xz'
   depends_on 'ossp-uuid'
-  depends_on 'pcre'
   depends_on 'rpm2cpio' => :build
 
   def install
+    # only rpm should go into HOMEBREW_CELLAR, not rpms built
+    inreplace "macros/macros.in", '@prefix@', HOMEBREW_PREFIX
     args = %W[
         --prefix=#{prefix}
         --localstatedir=#{var}
         --with-path-cfg=#{etc}/rpm
         --with-path-magic=#{HOMEBREW_PREFIX}/share/misc/magic
-        --with-extra-path-macros=#{lib}/rpm/macros.*
+        --with-path-sources=#{HOMEBREW_PREFIX}/src/rpm
```

Comment:
It can be moved, just that by default it has been `/usr/src/rpm` for 10+ years ("translated" to `/usr/local/src/rpm` on darwin) and in `/usr/src/redhat` long before that. On systems that don't "relax" their security permissions (like homebrew), this meant you had to build as root unless you modified the `%_topdir` macro first. As of lately, Fedora started putting it under the $HOME dir.

* http://rpm5.org/docs/max-rpm.html#s1-rpm-build-creating-build-directories
* http://rpm5.org/docs/max-rpm.html#s1-rpm-anywhere-different-build-area

MacPorts changed their tree (porthier) to include a "src" directory (just for rpmbuild, back when MacPorts still used rpm packages) in `/opt/local/src`, but I suppose that Homebrew can patch it...